src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -144,20 +146,36 @@ public static WebElement locateElementByXPath(String xpath) {
   }
 
   public static boolean isElementExists(String cssSelector) {
-    return SeleniumDriver.getDriver()
-      .findElements(By.cssSelector(cssSelector)).size() > 0;
+    try {
```

Comment:
These changes significantly reduce the time it takes to check for an element existence on the page. `findElements` won't stop until it finds `All` matching elements which we don't need. The happy path is now almost instant, the sad path takes about the same as `findElements`.